core/command.go

```diff
@@ -6,10 +6,94 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/peak/s5cmd/objurl"
 	"github.com/peak/s5cmd/op"
 	"github.com/peak/s5cmd/opt"
+	"github.com/peak/s5cmd/storage"
 )
 
+// Command is the representation of the command.
+type Command struct {
+	sourceDesc string
```

Comment:
`sourceDesc` sounds like it's related with the `src` description, but it's actually raw command string, right? Could we rename this to more expressive one? `raw`, `original` etc. would work. If you have a better one, it's ok for me.

Reply:
Done.